Thinktecture.Relay.OnPremiseConnector/SignalR/RelayServerConnection.cs

```diff
@@ -28,19 +36,14 @@ internal class RelayServerConnection : Connection, IRelayServerConnection
 		private readonly ConcurrentDictionary<string, IOnPremiseTargetConnector> _connectors;
 		private readonly HttpClient _httpClient;
 		private readonly int _relayServerConnectionId;
+
 		private CancellationTokenSource _cts;
 		private DateTime _lastHeartbeat;
 		private string _accessToken;
 		private bool _stopRequested;
 
-		private static int _nextId;
-		private static readonly Random _random = new Random();
-
-		private const int _MIN_WAIT_TIME_IN_SECONDS = 2;
-		private const int _MAX_WAIT_TIME_IN_SECONDS = 30;
-
 		public RelayServerConnection(string userName, string password, Uri relayServerUri, int requestTimeout, IOnPremiseTargetConnectorFactory onPremiseTargetConnectorFactory, ILogger logger)
-			: base(new Uri(relayServerUri, "/signalr").AbsoluteUri, "version=2")
+			: base(new Uri(relayServerUri, "/signalr").AbsoluteUri, $"version={CONNECTOR_VERSION}&assemblyVersion={Assembly.GetEntryAssembly().GetName().Version}")
```

Comment:
Let's change the parameter names to `cv` for connector version and `av` for assembly version.

Reply:
done